### PR TITLE
code to fix failures caused due to consul acl default policy deny

### LINKED ISSUE #19: Fix - ACL problems consul_acl_default_policy=deny
fredrikhgrelland/vagrant-hashistack#344

### PR DESCRIPTION
**Pull request by Neha-Sinha2305**
closes #19

### CODE REVIEW

**Review by zhenik — CHANGES_REQUESTED** [review]
LGTM

**Review by zhenik — APPROVED** [review]

File: Vagrantfile.default [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Vagrant.configure("2") do |config|
     config.vm.box = "fredrikhgrelland/hashistack"
     config.vm.box_version = ">= 0.4, < 0.5"
+    config.vm.boot_timeout = 500
```
Comment (claesgill) [inline Vagrantfile.default:4]: Should this be done also in the template?

**Review by claesgill — APPROVED** [review]
LGTM

File: Vagrantfile.default (hunk 1 above)
Comment (Neha-Sinha2305) [reply Vagrantfile.default:4]: May be..I added this as the tests failed with following error message:
```
If you look above, you should be able to see the error(s) that
Vagrant had when attempting to connect to the machine. These errors
are usually good hints as to what may be wrong.

If you're using a custom box, make sure that networking is properly
working and you're able to connect to the machine. It is a common
problem that networking isn't setup properly in these boxes.
Verify that authentication configurations are also setup properly,
as well.

If the box appears to be booting properly, you may want to increase
the timeout ("config.vm.boot_timeout") value.
make: *** [up] Error 1
Error: Process completed with exit code 2.
```
The default timeout value is 300 seconds : https://www.vagrantup.com/docs/vagrantfile/machine_settings#config-vm-boot_timeout
I will create an issue so we can keep a track of this.

**Pull request merged**